typescript-timers/interval.ts

```diff
@@ -0,0 +1,16 @@
+const $countdownDisplay = document.getElementById('countdown-display');
+if (!$countdownDisplay) throw new Error('$countdownDisplay query has failed');
+
+let count = 4;
+
+$countdownDisplay.textContent = count.toString();
+
```

Comment:
Your `$countDownDisplay` element already has a starting `textContent` of 4 so this does not do anything.